components/user/UserTabs.js

```diff
@@ -93,7 +93,7 @@ export default function UserTabs({ tabs, setTabs, setUserData, userData }) {
           name="tabs"
           onChange={changeTab}
           className="block w-full rounded-md border-gray-300 focus:border-indigo-500 focus:ring-indigo-500"
-          defaultValue={tabs.find((tab) => tab.current).name}
+          defaultValue={tabs.find((tab) => tab.current.name)}
```

Comment:
Confused about the change in this component as the PR is only for updating your profile.  Should this be here?

Reply:
I'll separate this in the new PR.